src/hotspot/cpu/aarch64/templateTable_aarch64.cpp

```diff
@@ -3242,68 +3288,29 @@ void TemplateTable::fast_xaccess(TosState state)
 //-----------------------------------------------------------------------------
 // Calls
 
-void TemplateTable::prepare_invoke(int byte_no,
-                                   Register method, // linked method (or i-klass)
-                                   Register index,  // itable index, MethodType, etc.
-                                   Register recv,   // if caller wants to see it
+void TemplateTable::prepare_invoke(Register recv,   // if caller wants to see it
                                    Register flags   // if caller wants to test it
```

Comment:
Hi All, 

I think `flags` argument here is unnecessary, as it is not being in this method. 

similarly `invokestatic` & `invokespecial` method is getting flags value from `load_resolved_method_entry_special_or_static` but it's not using it.